timemachine/potentials/jax_utils.py

```diff
@@ -8,7 +8,7 @@
 
 def convert_to_4d(x3, lamb, lambda_plane_idxs, lambda_offset_idxs, cutoff):
     # d4 = lambda_to_w(lamb, lambda_plane_idxs, lambda_offset_idxs, cutoff)
-    d4 = cutoff*lambda_plane_idxs + lambda_offset_idxs*lamb
```

Comment:
@proteneer : Could you please add a line comment on what this updated conversion function should do?